traits/util/trait_documenter.py

```diff
@@ -131,15 +131,35 @@ def _get_trait_definition(self):
                 name_found = True
 
         # Retrieve the trait definition.
-        definition_tokens = []
-        for type, name, start, stop, line in tokens:
-            if type == token.NEWLINE:
-                break
-            item = (type, name, (1, start[1]), (1, stop[1]), line)
-            definition_tokens.append(item)
-
+        definition_tokens = _get_definition_tokens(tokens)
         return tokenize.untokenize(definition_tokens).strip()
 
+def _get_definition_tokens(tokens):
+    """ Given the tokens, extracts the definition tokens.
```

Comment:
`tokens` is an iterator that has been forwarded at the definition token of the Trait. I would suggest to updated the docstring.